src/main/java/com/google/api/gax/grpc/InstantiatingChannelProvider.java

```diff
@@ -159,6 +150,22 @@ public boolean shouldAutoClose() {
   }
 
   @VisibleForTesting
+  public String serviceHeader() {
+    String grpcVersion = ManagedChannel.class.getPackage().getImplementationVersion();
+    if (grpcVersion == null) {
+      grpcVersion = DEFAULT_GRPC_VERSION;
+    }
```

Comment:
Nit: perhaps the above should be a private `getGrpcVersion()` method, like `getGaxVersion()`.